source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -0,0 +1,49 @@
+---
+title: Prefilling fields on the payment page
+weight: 137
+---
+
+# Prefilling fields on the payment page
+
+If you collect a user's details in your service before you redirect them to GOV.UK Pay, you can prefill some of the fields on the __Enter card details__ page. If you prefill fields, users should not need to enter their details twice.
+
+Users can still edit these details before they complete their payment.
+
+To prefill fields, include the following optional parameters in the <a href="https://govukpay-api-browser.cloudapps.digital/#newpayment" target="blank">API request</a> when you [create a new payment](/payment_flow/#making-a-payment):
+
+- `email`
+- `prefilled_cardholder_details` - with optional `cardholder_name` and `billing_address` parameters
+
+For example:
+
+```javascript
+"email": "sherlock.holmes@example.com",
+"prefilled_cardholder_details": {
+    "cardholder_name": "Sherlock Holmes",
+    "billing_address": {
+        "line1": "221 Baker Street",
+        "line2": "Flat b",
+        "postcode": "NW1 6XE",
+        "city": "London",
+        "country": "GB"
+ }
+```
+
+All the parameters in `billing_address` are optional.
+
+If you include the `country` parameter, it must be an <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'Great Britain'.
+
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more information.
+
+## Data collected by your GOV.UK Pay admin account
+
+After the user completes their payment, your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) will collect the user's:
+
+- email address - even if you've disabled collecting your users' email addresses
+- cardholder name
+- billing address
+
+If the user does not complete their payment, your GOV.UK Pay admin account will still collect the billing address. This will happen even if you've chosen not to collect your users' billing addresses.
```

Comment:
We don't collect it, we store it if it is sent to us. We don't display the billing address fields if the service decide that we shouldn't collect them. I'm not sure how this should be in the document...

Reply:
Thanks Sandor - I've tweaked in https://github.com/alphagov/pay-tech-docs/pull/286/commits/932f05e0559a8b7ee8de1faf4eaab90b28436355.